CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [3.98.0] - 2020-01-02
 ### Added
+- `savingsAsPercentage` prop
```

Comment:
The PR changes should be under `[Unreleased]`.

Example:

```md
## [Unreleased]

### Added
- `savingsAsPercentage` prop
```

Reply:
Like so?
```
## [Unreleased]

### Added
  - `savingsAsPercentage` prop

## [3.98.0] - 2020-01-02
### Added
- `showValueForVariation` to `SKUSelector`.
```

Reply:
Yes, just mind the extra spaces before the `-`

Reply:
Done!